src/Android/Xamarin.Android/groups.json

```diff
@@ -217,6 +217,23 @@
       ]
     },
     {
+      "Name": "Scenes",
+      "CategoryName": "Scenes",
+      "SubCategories": [
+        {
+          "Name": "Scene symbols",
```

Comment:
I think we need to have think about where we put Scene samples as it could well become a catch bucket full of various types of samples. this is cool for now though.